packages/@react-spectrum/breadcrumbs/test/Breadcrumbs.test.js

```diff
@@ -114,13 +125,15 @@ describe('Breadcrumbs', function () {
 
   it('Handles max visible items', () => {
     let {getByText, getByRole} = render(
-      <Breadcrumbs maxVisibleItems="3" >
-        <BreadcrumbItem >Folder 1</BreadcrumbItem>
-        <BreadcrumbItem >Folder 2</BreadcrumbItem>
-        <BreadcrumbItem >Folder 3</BreadcrumbItem>
-        <BreadcrumbItem >Folder 4</BreadcrumbItem>
-        <BreadcrumbItem >Folder 5</BreadcrumbItem>
-      </Breadcrumbs>
+      <Provider theme={theme}>
+        <Breadcrumbs maxVisibleItems="3" >
+          <Item >Folder 1</Item>
```

Comment:
Just a style thing, you should be able to remove the whitespace before the closing bracket for all the items in this file

Reply:
<Item> instead of <Item >